webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue

```diff
@@ -344,14 +344,7 @@ export default {
     postMessage() {
       // Using a ref to the editor component and the getMessage method is mandatory to
       // be sure to get the most up to date value of the message
-      let message = this.ckEditorInstance.getMessage();
-      const tempdiv = $('<div class=\'temp\'/>').html(message || '');
-      tempdiv.find('[data-atwho-at-value]')
-        .each(function() {
-          const value = $(this).attr('data-atwho-at-value');
-          const pattern = new RegExp(`<span class="atwho-inserted" data-atwho-at-query="[^"]*" data-atwho-at-value="${value}"[^>]*>(.*?)</span> </span>`, 'g');
-          message = message.replace(pattern, value ? `@${value}` : '');
-        });
+      const message = this.ckEditorInstance.updateMessageBeforPost();
```

Comment:
You may need to make the call of `updateMessageBeforPost` inside the method `getContentToSave` of `RichEditor.vue`
```suggestion
```

Reply:
if I move updateMessageBeforPost to getContentToSave, I will lose the initial message containing the invalid users, and I need to keep this message with the valid and invalid users for the connected user to decide to change the audience